src/actors/mod.rs

```diff
@@ -1,5 +1,6 @@
 //! Helper actors
 
+#[cfg(feature = "mocker")]
```

Comment:
Had to disable Mocker. Otherwise the compiler complained about conflicting implementations of `Handler`.